scanpy/tests/conftest.py

```diff
@@ -9,7 +9,7 @@
 import pytest
 
 import scanpy
-
+from scanpy.tests.fixtures import array_type, float_dtype  # noqa: F401
```

Comment:
It’s better to import fixtures in the conftest file, or use plugins, than to import fixtures in the test file

Reply:
Why?

Reply:
Let’s meet up and we can go over pytest’s namespacing and so on. I think that’s more effective than picking out one aspect here.